eZ/Publish/Core/Search/Legacy/Content/WordIndexer/Gateway/DoctrineDatabase.php

```diff
@@ -140,7 +140,7 @@ public function index(FullTextData $fullTextData)
                 ++$wordCount;
                 // if we have "www." before word than
                 // treat it as url and add additional entry to the index
-                if (mb_substr(mb_strtolower($word), 0, 4) === 'www.') {
+                if (mb_strtolower(mb_substr($word, 0, 4)) === 'www.') {
```

Comment:
That's a very nice catch :)

Reply:
Indeed =)

Reply:
BTW, it's one of the inspections in Php Inspections (EA Extended). What's cool is that all changes made automatically (quick-fixes).